### PR TITLE
Create directory for nexus-data on node (i.e. ec2) startup

### PR DESCRIPTION
**Pull request by bobotimi**
Adds daemonset  and config map that'll create a directory for the nexus-data to be mounted into on when a new EC2 starts up

### CODE REVIEW

File: nxrm-aws-resiliency/templates/workdir-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-{{ .Chart.Version }}-{{ .Release.Name }}-create-nexus-workdir-config
```
Comment (Mykyta) [inline nxrm-aws-resiliency/templates/workdir-configmap.yaml:4]: as far as I know, this name must be no more than 63 characters. 
So `nxrm-aws-resiliency-45.1.0-<Release.Name>-create-nexus-workdir-config-?????????` contains more than 63 symbols event without `Release.Name`.

File: nxrm-aws-resiliency/templates/workdir-daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    metadata:
+      labels:
+        job: dircreator
+    spec:
+      hostPID: true
+      restartPolicy: Always
+      initContainers:
+        # Copy file for creating nexus work directory over and execute it on host 
+        - name: create-nexus-work-dir
+          image: ubuntu:latest
```
Comment (Mykyta) [inline nxrm-aws-resiliency/templates/workdir-daemonset.yaml:20]: for the docker image, it is not recommended to use `latest` tag

File: nxrm-aws-resiliency/templates/workdir-configmap.yaml (hunk 1 above)
Comment (bobotimi) [reply nxrm-aws-resiliency/templates/workdir-configmap.yaml]: https://github.com/sonatype/nxrm3-helm-repository/pull/41/commits/35fb1119fb025a37ef8acd8ed029b7d3fa06d55e

File: nxrm-aws-resiliency/templates/workdir-daemonset.yaml (hunk 2 above)
Comment (bobotimi) [reply nxrm-aws-resiliency/templates/workdir-daemonset.yaml]: https://github.com/sonatype/nxrm3-helm-repository/pull/41/commits/35fb1119fb025a37ef8acd8ed029b7d3fa06d55e

**Review by Mykyta — APPROVED** [review]

**Review by oleksiirudyk — APPROVED** [review]

**Review by oleksiirudyk — APPROVED** [review]

**Pull request merged**